subprojects/provider/src/main/kotlin/org/gradle/kotlin/dsl/provider/KotlinScriptEvaluator.kt

```diff
@@ -255,3 +245,13 @@ private
 val embeddedKotlinModules by lazy {
     transitiveClosureOf("stdlib-jdk8", "reflect")
 }
+
+
+private
+inline fun ClassPathModeExceptionCollector.ignoringErrors(action: () -> Unit) =
+    try {
+        action()
+    } catch (e: Exception) {
+        e.printStackTrace()
```

Comment:
Use a logger?

Reply:
This codepath only happen through the IntelliJ script dependencies resolver, which will log those collected exceptions. No need to print the trace here.

:x: remove that line